src/GitHub.VisualStudio/Views/GitHubPane/PullRequestDetailView.xaml

```diff
@@ -245,6 +247,25 @@
                     </ItemsControl>
                 </ghfvs:SectionControl>
 
+                <ghfvs:SectionControl Name="checksSection"
+                                      HeaderText="Checks"
+                                      IsExpanded="True"
+                                      Margin="0 8 0 0"
+                                      ghfvs:ScrollingVerticalStackPanel.IsFixed="true">
+                    <ItemsControl ItemsSource="{Binding Checks}" Margin="0 4 12 4">
+                        <ItemsControl.ItemsPanel>
+                            <ItemsPanelTemplate>
+                                <StackPanel Grid.IsSharedSizeScope="True" />
+                            </ItemsPanelTemplate>
+                        </ItemsControl.ItemsPanel>
+                        <ItemsControl.ItemTemplate>
```

Comment:
I don't think this `ItemTemplate` should be needed. The `ViewLocator` should automatically find the relevant view.